COVERAGE.md

```diff
@@ -0,0 +1,73 @@
+# Coverage reports
+
+## Caveat
+
+Until each module can be migrated to support Node.js's builtin ESM
+implementation (`NESM`), the coverage line numbers will be out-of-sync with
+reality.
+
+In addition, we will have to implement source maps in all of our
+source-to-source transforms (such as `@agoric/bundle-source`,
+`@agoric/transform-metering`, and `@agoric/static-module-record`).
+
+## Reports
+
+Coverage reports for the current main branch (whose packages support `NESM`) are
+published by CI to: https://agoric-sdk-coverage.netlify.app
+
+You can create a report in any package (including the top-level directory):
+
+```sh
+# Get options available for coverage:
+yarn c8 --help
+# Run ava under Node.js coverage and display a summary:
+yarn c8 -a ava
+# Generate a nice, detailed HTML report:
+yarn c8 report --reporter=html-spa
+open coverage/html/index.html
+```
+
+## NESM Support
```

Comment:
Let’s spell this “Node.js ESM” formally.

Reply:
Done.